eogrow/core/area/custom_grid.py

```diff
@@ -81,3 +81,46 @@ def _construct_file_path(self, *, prefix: str, suffix: str = "gpkg", **_: Any) -
         filename = filename.replace(" ", "_")
 
         return fs.path.combine(self.storage.get_cache_folder(), f"{filename}.{suffix}")
+
+
+class NewCustomGridAreaManager(BaseAreaManager):
+    """Area manager that works with a pre-defined grid of EOPatches"""
+
+    class Schema(BaseAreaManager.Schema):
+        grid_folder_key: str = Field("input_data", description="Which folder the grid file is in.")
+        grid_filename: Path = Field(
+            description=(
+                "A Geopackage with a collection of bounding boxes and attributes that will define EOPatches. In"
+                " case bounding boxes are in multiple CRS then each Geopackage layer should contain bounding boxes"
+                " from one CRS."
```

Comment:
Just checking, is it really possible to put layers with different CRS to the same geopackage?